### ISSUE TITLE
{var} ? ""-"" <- broke

### ISSUE THREAD

**Issue by Darkweasam** [SkriptLang/Skript#4192]
### Skript/Server Version

```
2.6-beta2
Paper104 1.17.1-r0.1snapshot
```

### Bug Description

Can't understand this expression: {_test} ? ""-""

### Expected Behavior

It should return a dash if its not set ( being <none> )

### Steps to Reproduce

command /test:
trigger:
send "%{_test} ? ""-""%"

### Errors or Screenshots

```
Replace this text with your paste link or errors/screenshots/videos.
```

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by TheRealUnderscore** [comment]
Quote parsing was fixed in 2.6-alpha1, use `? "-"`

**Comment by Darkweasam** [comment]
> 
> 
> Quote parsing was fixed in 2.6-alpha1, use `? "-"`

Thank you, this fixed my issue!
